app/javascript/components/my-studies/StudyUsageInfo.jsx

```diff
@@ -0,0 +1,129 @@
+  useEffect(() => {
+    fetchStudyUsage(study._id.$oid).then(response => {
+      setUsageInfo(response)
+      setIsLoading(false)
+    })
+  }, [study.accession])
+
+  const tableData = []
+
+  if (usageInfo) {
+    // generate the trace information and table information
+    const viewData = usageInfo.pageViews.data
+    const viewsTrace = {
+      type: 'bar',
+      x: viewData.series,
+      y: viewData.series.map(x => {
+        return viewData.values[x] ? viewData.values[x] : 0
+      })
+    }
+    const viewsLayout = {
+      xaxis: { title: 'Month' },
+      yaxis: { title: 'Unique users' }
+    }
+    Plotly.react(viewsGraphElementId, [viewsTrace], viewsLayout)
+    tableData.push(viewsTrace.x)
+    tableData.push(viewsTrace.y)
+
+    // sum together the single file clicks and the bulk downloads for the graph
+    const downloadData = usageInfo.fileDownloads.data
+    const downloadTableData = []
+    const bulkDownloadData = usageInfo.bulkDownloads.data
+    const bulkDownloadTableData = []
+    const downloadsTrace = {
+      type: 'bar',
+      x: downloadData.series,
+      y: downloadData.series.map(x => {
+        const singleFileDownloads = downloadData.values[x] ? downloadData.values[x] : 0
+        downloadTableData.push(singleFileDownloads)
+        const bulkFileDownloads = bulkDownloadData.values[x] ? bulkDownloadData.values[x] : 0
+        bulkDownloadTableData.push(bulkFileDownloads)
+        return singleFileDownloads + bulkFileDownloads
+      })
+    }
+    const downloadsLayout = {
+      xaxis: { title: 'Month' },
+      yaxis: { title: 'Download events' }
+    }
+    Plotly.react(downloadsGraphElementId, [downloadsTrace], downloadsLayout)
+    tableData.push(downloadTableData)
+    tableData.push(bulkDownloadTableData)
+  }
+
+  return <div>
+    <h3>Study statistics</h3>
+    {studyAccession}: <a href={`/single_cell/study/${studyAccession}`}>{ study.name }</a><br/>
+    <div className="text-center">
+      Users per month who viewed this study. <InfoPopup content="Unique users per month visiting the study overview page (i.e. each user only counts at most once per month)"/><br/>
+      <LoadingSpinner isLoading={isLoading} testId="study-usage-spinner"/>
+    </div>
+    <div
+      className="scatter-graph"
+      id={viewsGraphElementId}
+      data-testid={viewsGraphElementId}
+    ></div>
+    <br/><br/>
+    <div className="text-center">
+      File download events <InfoPopup content="Clicks on a single file-download button, or a bulk download that included this study"/><br/>
+      <LoadingSpinner isLoading={isLoading}/>
+    </div>
+    <div
+      className="scatter-graph"
+      id={downloadsGraphElementId}
+      data-testid={viewsGraphElementId}
+    ></div>
+    <br/><br/>
+    <div className="form-terra">
+      <h5 onClick={() => setShowTable(!showTable)}>
+        <span className="action">
+          { showTable ? <FontAwesomeIcon icon={faChevronUp}/> : <FontAwesomeIcon icon={faChevronDown}/> }
```

Comment:
Non-blocking: Traditionally, I think the icons for "closed" is `faChevronRight` and "open" is `faChevronDown`.  I found the "up" chevron confusing there.

Reply:
I don't have a strong opinion on this, and there's nothing in the Terra Style guide (https://www.sketch.com/s/cca3a834-985b-4165-bbae-21f615aebd2d) about it.  I just pulled this from the Upload Wizard design from Jerome.  Totally open to changing both, as long as we're consistent